tests/test-module-test.js

```diff
@@ -94,13 +94,43 @@ moduleFor('component:x-foo', 'component:x-foo -- callback context', {
     setupRegistry();
   },
 
+  setup: function() {
+    originalDeprecate = Ember.deprecate;
+  },
+
+  teardown: function() {
+    Ember.deprecate = originalDeprecate;
+  },
+
   getSubjectName: function() {
     return this.subjectName;
+  },
+
+  getFoo: function() {
+    return this.foo;
   }
 });
 
-test("callbacks are called in the context of the module", function() {
+test("can access TestModule properties from a callback but raises a deprecation", function() {
+  var deprecations = [];
+  Ember.deprecate = function(message) {
+    deprecations.push(message);
+  };
+
   equal(this.getSubjectName(), 'component:x-foo');
+  ok(Ember.A(deprecations).contains('Accessing the test module property "subjectName" from a callback is deprecated.'));
+});
+
+test("can access test context properties from a callback's 'this' and not raise a deprecation", function() {
+  var deprecations = [];
+  Ember.deprecate = function(message) {
+    deprecations.push(message);
+  };
+
+  this.foo = 'bar';
+
+  equal(this.getFoo(), 'bar');
+  ok(!deprecations.length);
```

Comment:
I am unsure why, but this assertion is failing on ember-1.13.  I am rerunning in Travis in case it is a fluke.